.stylelintrc

```diff
@@ -12,6 +12,12 @@
         "ignore": ["custom-elements"]
       }
     ],
+    "selector-pseudo-element-no-unknown": [
+      true,
+      {
+        "ignorePseudoElements": ["ng-deep"]
+      }
+    ],
```

Comment:
Is it recommended to use ng-deep? If not, I would suggest not to ignore it instead use a bypass flag and add code-comment explaining why `ng-deep` used.

Reply:
done